### ISSUE TITLE
Accessing optional nested struct value

### ISSUE THREAD

**Issue by silentninja** [schibsted/layout#126]
I am trying to access a nested struct value using dot notation. 
Something like this 
```
// Model
class NestedModel {
var name: String?;
}
class Model {
var nestedModel: NestedModel?;
}
// layout.xml
    <UILabel
        width="100%"
        textColor="titleColor"
        text="{nestedModel.name}"
    />
```
The error says it is an unknown property.
```
 Unknown property name in nestedModel in nestedModel.name expression in text expression in UILabel in UIView
```
Is there any way to unwrap the optional value in the template

**Comment by nicklockwood** [comment]
This looks like an oversight in Layout’s type system. I don’t think there’s a workaround other than making the value non-optional - I’ll try to fix it for the next release.

**Comment by nicklockwood** [comment]
@silentninja this is now fixed on the develop branch.

**Comment by nicklockwood** [comment]
Fixed in version 0.6.25
